### DISCUSSION TITLE
how do i hide devices that i am not currently using?

### DISCUSSION THREAD

**Discussion by alfredonevv** [File-New-Project/EarTrumpet#1111] [Q&A] [answered]
in the past when i would click the sound icon i was shown the sliders for the current device i was using. now when i click the icon i see the sliders for all audio devices. how do i make it so i can only see the active audio device?

note: i switch frequently between speakers and my headset, so i don't want any of the devices disabled in the windows sounds, i just want to be able to see one set of sliders at a time.
![image](https://user-images.githubusercontent.com/103166585/162123819-30939f3a-376f-4ad8-be18-6fc4ee6bbc00.png)

**Comment by riverar** [comment, marked as answer]
Click the chevron/arrow at the top (picture below). 

![image](https://user-images.githubusercontent.com/475132/162126242-fd83ec69-b305-4d1a-895d-cc42def0cc65.png)
